spec/controllers/v1/root_controller_spec.rb

```diff
@@ -1,8 +1,7 @@
-RSpec.describe Api::V1x0::RootController, :type => :request do
+RSpec.describe Api::V1x0::RootController, :type => [:request, :v1x1] do
```

Comment:
Why here `Api::V1x0` does not match `[:request, :v1x1]`?

Reply:
I moved that root controller to a major version only path.  Because that spec always should return the highest minor version.